docs/cli.rst

```diff
@@ -211,6 +211,17 @@ The VHDL revision can be specified through the :ref:`python_interface`
 Alternatively, environment variable ``VUNIT_VHDL_STANDARD`` can be set to
 ``1993``, ``2002``, ``2008`` or ``2019``.
 
+.. note:: ``93``, ``02``, ``08`` or ``19`` can be also used.
+
+.. note:: ``2008`` is the default.
+
+.. note:: you will receive an ERROR if you specify a not supported revision
+          (*"unknown"* when it is not supported by Vunit and *"invalid"* when
+          it is not supported by a particular tool).
+
+.. warning:: the ``context`` statement is not supported previous 2008
```

Comment:
Similarly to the previous comment, I don't think this is particular to this section. That is one of the multiple differences introduced in VHDL 2008. The point is that the user guide and the examples are expected to be used with simulators having a "decent" VHDL 2008 support.

I do agree it would be interesting to have some comment about how some of the examples can be "easily" modified to work on specific simulators which support some VHDL 2008 but are lacking others. However, that belongs to section Examples.

Reply:
An explanation in the examples sounds good. So, a link here to such a section would be enough.